kustomize/resources.yaml

```
@@ -10,6 +10,29 @@ metadata:
 spec:
   template:
     spec:
+      initContainers:
+        - name: template-quorum-key
+          image: ghcr.io/tkhq/frames
+          env:
+            - name: TURNKEY_SIGNER_ENVIRONMENT
+              value: "prod"
+          command:
+            - sh
+            - -c
+            - |
+                mkdir -p templated/import && envsubst '${TURNKEY_SIGNER_ENVIRONMENT}' < import/index.template.html > templated/import/index.html;
+                mkdir -p templated/import && envsubst '${TURNKEY_SIGNER_ENVIRONMENT}' < import/standalone.template.html > templated/import/standalone.html;
+                mkdir -p templated/export && envsubst '${TURNKEY_SIGNER_ENVIRONMENT}' < export/index.template.html > templated/export/index.html;
```

Can do all the `mkdir` upfront
```suggestion
            - |
                mkdir -p templated/export templated/import;
                envsubst '${TURNKEY_SIGNER_ENVIRONMENT}' < import/index.template.html > templated/import/index.html;
                envsubst '${TURNKEY_SIGNER_ENVIRONMENT}' < import/standalone.template.html > templated/import/standalone.html;
                envsubst '${TURNKEY_SIGNER_ENVIRONMENT}' < export/index.template.html > templated/export/index.html;
```